nuxeo-services/nuxeo-platform-login-shibboleth/src/main/java/org/nuxeo/ecm/platform/shibboleth/service/ShibbolethAuthenticationServiceImpl.java

```diff
@@ -35,6 +35,11 @@
 
 public class ShibbolethAuthenticationServiceImpl extends DefaultComponent implements ShibbolethAuthenticationService {
 
+    /**
+     * @since 8.4
+     */
+    private static final String REDIRECT_URL = "redirect_url";
```

Comment:
Why initializing REDIRECT_URL with "redirect_url"? Is it a wanted fallback redirect url?